source/configuring-ember/specifying-url-type.md

```diff
@@ -13,30 +21,22 @@ Router.map(function() {
 });
 ```
 
+## history
```

Comment:
Maybe slightly out of scope so feel free to punt: History API is supported in everything except IE9 and JSBins, and thus I think it should be the first item in this list. `hash` should be portrayed as a fallback, not as the most important (first mentioned) kind of URL for an Ember app.

Good direction for a followup perhaps.